src/patterns/atoms/forms/radio-buttons.hbs

```diff
@@ -4,6 +4,8 @@ order: 4
 tips: |
   - Wrap the `label` element around both the control and its text label. This gives a bigger target for clicking.
   - Use the `mzp-u-inline` class to override the base block-level label styling.
+links:
+  See the choices component for styled radio buttons: /demos/forms/choices.html
```

Comment:
Same here, link to /patterns/molecules/forms.html#custom-checkboxes-and-radio-buttons